src/styles/layout.scss

```
@@ -22,7 +22,7 @@
   --container-sm: 560px; // narrow: forms, dialogs
   --container-md: 768px; // standard content column
   --container-lg: 1040px; // wide: two-column detail pages
-  --container-xl: 1280px; // max page width
+  --container-xl: 1680x; // max page width
```

Stylelint (17.6.0)

[error] Unexpected unknown unit "x" (unit-no-unknown)

(unit-no-unknown)